desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -3161,49 +3158,28 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
         };
         function upload() {
           var fd = new FormData();
-          fd.append('fileFormat', ko.mapping.toJSON(viewModel.createWizard.source));
+          var files = $('#inputfile')[0].files[0];
+          fd.append('inputfile', files);
+          var file_size = files.size;
+          if (file_size < 30*1000) {
```

Comment:
Tested for small files so, doing good
<img width="1790" alt="Screenshot 2021-04-07 at 11 28 25 PM" src="https://user-images.githubusercontent.com/36241930/113912862-411cf600-97f9-11eb-9508-7e03dbb92ced.png">
looking for data_size is less than the 2.5 MB(django.conf.global_settings have this value) then why it is showing error 

but when I am uploading the file of size 800KB then it is showing error (in ss).